.github/workflows/test.yaml

```diff
@@ -0,0 +1,18 @@
+name: test
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ['3.6', '3.7', '3.8', '3.9']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - run: pip install -r requirements.txt pytest
+      - run: python -m pytest beanprice
```

Comment:
beanprice?

Reply:
Ah, sorry, I am also working on beanprice a little and I mixed them here.